nautobot/extras/datasources/git.py

```diff
@@ -75,13 +88,48 @@ def pull_git_repository_and_refresh_data(repository_pk, request, job_result_pk):
         job_result.save()
         return
```

Comment:
Let's make the return type self-consistent

```suggestion
        return job_result, None
```